logger.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"io"
+	"log"
+	"os"
+)
+
+// LogLevel represents custom type for configure
+type LogLevel int
```

Comment:
its not really used as logger though, more likely general CLI output... 

with that in mind, maybe we could call this `output` or something?

then we merely have quiet output (just errors) or not (full output) 🤔

Reply:
also:
- this dont need to be exported
- could probably use `iota`

Reply:
Yeah I think you're right.

One doubt about logger naming, is it something for all logger related? (file name, all the functions in logger.go)
Or are you referring to only LogLevel?

Reply:
everything in the logger.go file (including the filename itself)